ports/models.py

```diff
@@ -27,23 +18,31 @@ class Port(models.Model):
     platforms = models.TextField(default='')
     categories = models.ManyToManyField(Category, related_name='category', db_index=True)
     long_description = models.TextField(default='')
-    depends_extract = models.TextField(default='')
     version = models.CharField(max_length=100)
-    revision = models.TextField(default='')
-    maintainers = models.ManyToManyField(Maintainer, related_name='maintainer')
+    revision = models.IntegerField(default=2)
     closedmaintainer = models.BooleanField()
     name = models.CharField(max_length=100, db_index=True)
     license = models.CharField(max_length=100)
-    depends_lib = models.TextField(default='')
-    depends_build = models.TextField(default='')
-    installs_libs = models.TextField(default='')
-    depends_fetch = models.TextField(default='')
-    depends_run = models.TextField(default='')
-    conflicts = models.TextField(default='')
-    replaced_by = models.TextField(default='')
-    depends_test = models.TextField(default='')
-    depends_patch = models.TextField(default='')
-    subports = models.TextField(default='')
+    installs_libs = models.BooleanField(default=False)
+    replaced_by = models.CharField(max_length=100)
+
+    objects = PortManager()
+
+
+class Dependency(models.Model):
+    port_name = models.ForeignKey(Port, on_delete=models.CASCADE, related_name="dependent_port", db_index=True)
+    dependencies = models.ManyToManyField(Port, db_index=True)
+    type = models.CharField(max_length=100)
```

Comment:
This field should be indexed as well. And for the sake of consistency ideally referencing (with a foreign key) yet another table holding types of dependencies.